postcss.config.js

```diff
@@ -51,7 +51,7 @@ module.exports = ({
 						const filePath = packageParts.length > 2 ? packageParts.slice(2).join("/") : "index.css";
 
 						if (packageParts[1] === "tokens") {
-							return join(__dirname, packageParts[1], "dist", "css", filePath);
+							return join(__dirname, packageParts[1], filePath);
```

Comment:
Are we no longer using the dist version of this file? Was this a fix?

Reply:
This was a fix, the filepath includes the dist/css filepath already.